docs/en/revocation-lists.rst

```diff
@@ -212,142 +231,220 @@ The following HTTP Status Codes MUST be supported:
     * - **Status Code**
       - **Body**
       - **Description**
-    * - *204 No Content*
-      - 
-      - The Digital Credential has been successfully revoked.
+    * - *200 Created*
+      - Revocation Assertion Response
+      - The Revocation Assertion Response has been successfully created.
     * - *400 Bad Request*
       - Error code and description
-      - The issuer cannot fulfill the request because of invalid parameters.
-    * - *404 Not Found*
```

Comment:
Why HTTP status 404 Not Found was removed?

If the Credential doesn't exist we should rise an error to inform the wallet instance.
The Status Code 200 means that the credential is succesfully revoked by the Credential Issuer.